mne/minimum_norm/tests/test_inverse.py

```diff
@@ -289,8 +286,8 @@ def test_apply_inverse_operator():
     inv_op = prepare_inverse_operator(inverse_operator, nave=evoked.nave,
                                       lambda2=lambda2, method="MNE")
     stc2 = apply_inverse(evoked, inv_op, lambda2, "MNE")
-    assert_array_almost_equal(stc.data, stc2.data)
-    assert_array_almost_equal(stc.times, stc2.times)
+    assert_allclose(stc.data, stc2.data)
+    assert_allclose(stc.times, stc2.times)
```

Comment:
`allclose` is more strict than `almost_equal`, also at most other places we use `allclose` so I thought this would be better.